stable/tomcat/values.yaml

```diff
@@ -35,6 +37,24 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+env: |
```

Comment:
You shouldn't need these as embedded documents.  You can do the following to demonstrate intent in values file which is how this kind of thing is usually handled ( see the `annotations: {}` line in ingress )

```
env: []
  # - name: env
  #   value: test
```

Reply:
OK. I'll update accordingly.